ci-operator/step-registry/upi/aws/pre/upi-aws-pre-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: upi-aws-pre
+  steps:
+  - ref: ipi-install-rbac
+  - chain: ipi-conf-aws
+  - ref: upi-install-aws
+  - ref: ipi-install-times-collection
```

Comment:
I was thinking creating another folder for the chain and then calling just the ref(upi-install-aws) and the ipi-install-times-collection would have more files than just add two lines here but I see your concern where this may be a problem in the future with respect to modularity? I will do the former.

Reply:
The former involves a lot of extra folders(for the upi and proxy steps) that I think are redundant and I feel it's probably best to just add it here? Thoughts?